Library/Formula/deckcontrol.rb

```diff
@@ -0,0 +1,15 @@
+require 'formula'
+
+class Deckcontrol < Formula
+    homepage 'https://github.com/bavc/deckcontrol'
```

Comment:
fixed in https://github.com/dericed/homebrew/commit/b653bbf01739ade2fb7162ad28c4d663ecccb182, thanks for showing me `brew audit`